homework_10/CurrentTime.java

```diff
@@ -0,0 +1,11 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class CurrentTime {
+    public static String time() {
+        LocalDateTime localTime = LocalDateTime.now();
+        DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss");
```

Comment:
Instead of hardcoding `"HH:mm:ss"` please pass a parameter `String pattern` to your method